packages/runtime/src/helpers/config.ts

```diff
@@ -156,7 +160,21 @@ export const configureHandlerFunctions = async ({
         netlifyConfig.functions[functionName].included_files.push(`!${moduleRoot}/**/*`)
       }
     })
-  })
+  }
+
+  configureFunction(HANDLER_FUNCTION_NAME)
+  configureFunction(ODB_FUNCTION_NAME)
+  if (SPLIT_API_ROUTES) {
+    for (const apiRoute of apiRoutes) {
+      const { functionName, includedFiles } = apiRoute
+      // TODO: add all the nextRoot/wasm/excludedModules stuff from above
+      netlifyConfig.functions[functionName] ||= { included_files: [] }
+      netlifyConfig.functions[functionName].node_bundler = 'none' as any
```

Comment:
we'll need to wait for https://github.com/netlify/build/pull/4981 to merge, before we can remove this + see E2E run through.